google-ads-examples/src/main/java/com/google/ads/googleads/examples/remarketing/RemoveFeedItemAttributeValue.java

```diff
@@ -108,41 +113,56 @@ public static void main(String[] args) {
    * Runs the example.
    *
    * @param googleAdsClient the Google Ads API client.
-   * @param params the RemoveFeedItemAttributeValueParams for the example.
+   * @param customerId the client customer ID in which to create criterion.
```

Comment:
"...in which to create criterion." -> "the client customer ID."

Same applies on the other `@param customerId` javadoc lines in this PR.